about/index.html

```diff
@@ -48,32 +48,17 @@ <h1 class="page-title">
     <div class="markdown-body">
       <p>Learn more about this styleguide, particularly how it’s built and who maintains it.</p>
 
-<h3 id="what-and-why">What and why</h3>
-
-<p>Primer is GitHub’s internal CSS framework. It includes basic global styling for typography, small components like buttons and tabs, and our general guidelines for writing HTML and CSS. It’s been used internally at GitHub for years now.</p>
-
-<p>Open-sourcing Primer means sharing and learning with the community. There’s a lot we can improve upon in Primer, and help is always appreciated. While we don’t currently plan on building this out as a competitor to other front-end frameworks, we will occasionally add, remove, or modify things.</p>
-
-<h3 id="browser-support">Browser support</h3>
-
-<p>Primer currently supports Internet Explorer 9+ and the latest two versions of Chrome, Safari, and Firefox on OS X and Windows. While not a responsive or mobile-focused project, Mobile Safari and Chrome for Android should render just fine. Support for Linux-based browsers is not strictly guaranteed, but accounted for whenever possible.</p>
-
-<h3 id="future-updates">Future updates</h3>
+      <h3 id="what-and-why">What and why</h3>
 
-<p>See the <a href="/roadmap">roadmap</a> for a rough outline on what’s slated for future versions of Primer.</p>
+      <p>Primer is GitHub’s internal CSS framework. It includes basic global styling for typography, small components like buttons and tabs, and our general guidelines for writing HTML and CSS. It’s been used internally at GitHub for years now.</p>
 
-<h3 id="dependencies">Dependencies</h3>
+      <h3 id="future-updates">Future Updates</h3>
 
-<ul>
-  <li>Our styles are built with SCSS.</li>
-  <li>Bower is used to link GitHub assets, namely our <a href="http://octicons.github.com">Octicons</a> icon font and Markdown styles.</li>
-  <li>Autoprefixer is used to generate vendor-prefixed declarations and is handled via our Gruntfile.</li>
-  <li>A custom plugin for snippet and example combos, <code class="highlighter-rouge">example.rb</code>, was created as a fork of Jekyll’s built-in <code class="highlighter-rouge">highlight</code> plugin.</li>
-</ul>
+      <p>Primer is currently being overhauled. The upcoming version of Primer will add new styles and bring the existing styles up to date, improve the build process, and include new and improved documentation. This will be open-sourced and shared with the community when it's complete, currently slated to happen in mid-to-late 2017.</p>
```

Comment:
Not a big deal, but all the styles are already open sourced. You could re-phrase the last sentence to this:

"New styles will be open-sourced in the [Primer repo](https://github.com/primer/primer-css) and [on NPM](https://www.npmjs.com/package/primer-css) as we add them, and the new documentation site will be shared when it's complete,..."